src/hound/index/index.go

```diff
@@ -57,7 +59,7 @@ type ExcludedFile struct {
 }
 
 func (s Snapshot) Open() (*Index, error) {
-	return Open(string(s))
+	return Open(string(s), "")
```

Comment:
This would be problematic if we still used this notion of a Snapshot, but we don't. So I'm going to merge your change and then fix this in a separate change.